### DISCUSSION TITLE
How to disable automatic updates of import paths

### DISCUSSION THREAD

**Discussion by odashi** [microsoft/pylance-release#3610] [Q&A]
When I moved some Python files, VSCode (maybe this extension) tries to update other source files to correct the import paths of the moved file. This is usually useful, but sometimes the operation requires to take a long time.

For example, one of the workspaces I am currently working contains thousands of source files, and the operation typically took several minutes and other work was blocked until its completion (or manually cancelling the behavior).

Are there any configs to disable this feature?
I searched similar questions anyway, but couldn't find answers in a reasonable time.

**Comment by brettcannon** [comment]
There isn't a specific way to turn it off (that I know of), but if you would like such a feature it would be an ask on Pylance as they provide the move refactoring support (I will move this there).

**Reply by odashi** [reply]
Got it, thanks for your support!
